src/content/hf2p/readme.md

```diff
@@ -0,0 +1,38 @@
+<figure>
+  <a href="HaloOnline_ConceptArt.jpg">
+    <img src="HaloOnline_ConceptArt.jpg" alt="""/>
+  </a>
+  <figcaption>
+    <p>Halo Online's Concept Art</p>
+  </figcaption>
+</figure>
+
+**Halo Online** is a canceled instalment of the Halo game series and runs on a modified [Halo 3][h3] Engine. It was created by [Saber Interactive][saber] and initially released as part of a closed beta in Russia on the PC in 2015-16 by [343 Industries][343i]. In 2016 the game was canceled as a part of *Halo: The Master Chief Collection* on PC was being ported to Microsoft Windows alongside other halo titles in the collection.
```

Comment:
```suggestion
**Halo Online** is a canceled installment of the Halo game series and runs on a modified [Halo 3][h3] Engine. It was created by [Saber Interactive][saber] and initially released as part of a closed beta in Russia on the PC in 2015-16.
```

No evidence that was the reason it was canceled and arguable it was published by Saber Interactive not 343i, should rephrase that to high-light it was a collaboration but that Saber Interactive's platform was used.